include/inexor/vulkan-renderer/availability_checks.hpp

```diff
@@ -40,34 +32,30 @@ class AvailabilityChecksManager {
     ~AvailabilityChecksManager() = default;
 
     /// @brief Checks if a certain Vulkan instance layer is available on the system.
-    /// https://www.khronos.org/registry/vulkan/specs/1.1-extensions/man/html/vkEnumerateInstanceLayerProperties.html
     /// @param instance_layer_name The name of the Vulkan instance layer.
-    /// @return true if the Vulkan instance layer is available, false otherwise.
+    /// @return 'true' if the Vulkan instance layer is available, 'false' otherwise.
     /// @note Available instance layers can be enabled by passing them as parameter during Vulkan instance creation.
     [[nodiscard]] bool has_instance_layer(const std::string &instance_layer_name);
 
     /// @brief Checks if a certain Vulkan instance extension is available on the system.
-    /// https://www.khronos.org/registry/vulkan/specs/1.1-extensions/man/html/vkEnumerateInstanceExtensionProperties.html
     /// @param instance_extension_name The name of the Vulkan instance extension.
-    /// @return true if the Vulkan instance extension is available, false otherwise.
+    /// @return 'true' if the Vulkan instance extension is available, 'false' otherwise.
```

Comment:
```suggestion
    /// @return ``true`` if the Vulkan instance extension is available, ``false`` otherwise.
```

Reply:
Done. I removed the `false otherwise` everywhere.